cardano-api/src/Cardano/Api/Types.hs

```diff
@@ -86,15 +104,27 @@ type ByronTx              = Byron.TxAux
 type ByronTxId            = Byron.TxId
 type ByronWitness         = Byron.TxInWitness
 
-type ShelleyVerificationKey = Shelley.VKey   Shelley.TPraosStandardCrypto
-type ShelleySigningKey      = Shelley.SKey   Shelley.TPraosStandardCrypto
+type ShelleyCoin                   = Shelley.Coin
+type ShelleyVerificationKey        = Shelley.VKey   Shelley.TPraosStandardCrypto
+type ShelleyVerificationKeyHash    = Shelley.DiscKeyHash 'Regular Shelley.TPraosStandardCrypto
+type ShelleySigningKey             = Shelley.SKey   Shelley.TPraosStandardCrypto
+type ShelleyVRFVerificationKeyHash = Shelley.Hash HASH.Blake2b_256 (Shelley.VerKeyVRF VRF.SimpleVRF)
 type ShelleyAddress         = Shelley.Addr   Shelley.TPraosStandardCrypto
 type ShelleyTxIn            = Shelley.TxIn   Shelley.TPraosStandardCrypto
 type ShelleyTxOut           = Shelley.TxOut  Shelley.TPraosStandardCrypto
 type ShelleyTxBody          = Shelley.TxBody Shelley.TPraosStandardCrypto
 type ShelleyTx              = Shelley.Tx     Shelley.TPraosStandardCrypto
 type ShelleyTxId            = Shelley.TxId   Shelley.TPraosStandardCrypto
 type ShelleyWitnessVKey     = Shelley.WitVKey Shelley.TPraosStandardCrypto
```

Comment:
If we're keeping these all aligned, I guess we can do the same here.

Reply:
I was figuring out the best way to align as the diff will make it harder to see what was added. I'll make the change though.